specification/migrate/get_reindex_status/MigrateGetReindexStatusRequest.ts

```diff
@@ -21,8 +21,9 @@ import { RequestBase } from '@_types/Base'
 import { Indices } from '@_types/common'
 
 /**
- * This API returns the status of a migration reindex attempt for a data stream or index
+ * Get the reindexing status.
```

Comment:
Same here, if we have the space `Get the migration reindexing status` would be less ambiguous.

Reply:
Fixed in a1f8330f5fe05776ae8877c187104fe7920f79ef